CMakeLists.txt

```diff
@@ -17,7 +17,10 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 macro(enable_strict_compiler_warnings target)
   target_compile_options(${target} PRIVATE
     $<$<CXX_COMPILER_ID:MSVC>:/WX /W4>
-    $<$<NOT:$<CXX_COMPILER_ID:MSVC>>:-Wall -Wextra -Wpedantic -Werror -Wold-style-cast -Wfloat-equal>
```

Comment:
This conditional stuff is crazy – can we not just use regular `if()` to do this?

Reply:
I can change it to that, yes